site/package.json

```
@@ -99,6 +100,6 @@
     "eslint:fix": "eslint . --fix",
     "test": "jest",
     "test:api": "jest --config jest.config.api.js",
-    "build": "tsc -p . && node .build.js && vite build vue"
+    "build": "tsc -p . && vue-tsc --noEmit && node .build.js && vite build vue"
```

```suggestion
    "build": "vue-tsc --noEmit && node .build.js && vite build vue"
```
We can remove running `tsc` since `vue-tsc` will also check the typescript files

---

Ah, nice catch.